docs/tips-and-tricks.md

```diff
@@ -0,0 +1,157 @@
+
+    from clove.network import Litecoin
+
+    network = Litecoin()
+
+    utxo = network.get_utxo(address='LUAn5PWmsPavgz32mGkqsUuAKncftS37Jq', amount='0.01')
+
+    utxo
+    [
+        Utxo(
+            tx_id='0cd90567497823097d03464b4b2d08dd659f1c5621dd55e9540bc9bcd3e191ec',
+            vout='0',
+            value='0.00976168',
+            tx_script='76a91485c0522f6e23beb11cc3d066cd20ed732648a4e688ac',
+            wallet=None,
+            secret=None,
+            refund=False
+        ),
+        Utxo(
+            tx_id='a5c027027c695f403fe570850e35ffd44bb28479ecaaee039372015fe0aae7b2',
+            vout='0',
+            value='0.00097114',
+            tx_script='76a91485c0522f6e23beb11cc3d066cd20ed732648a4e688ac',
+            wallet=None,
+            secret=None,
+            refund=False
+        )
+    ]
+
+
+### Manual
+
+If some network doesn't have block explorer integration to gather UTXO automatically you can still create it manually by getting required data (`tx_id`, `vout`, `value` and `tx_script`) from some other source.
+
+    from clove.network.bitcoin.utxo import Utxo
+
+    utxo_list = [
+        Utxo(
+            tx_id='5d920e7093b2f0ac94cb1c13a42a79ed1c1290fcc4155d15a123d69b1afe05d2',
+            vout=1,
+            value=2.086,
+            tx_script='76a9142b6a3314e8fcf1f1fd6b4d70b112bd5a1928505788ac'
+        ),
+    ]
+
+and then you can use this list in atomic swap transaction.
+
+    network.atomic_swap(
+        sender_address=alice_address,
+        recipient_address=bob_address,
+        value=1.5,
+        solvable_utxo=utxo_list,
+    )
+
+
+## Getting fee
+
+In Bitcoin-base networks every transaction needs to include some small fee for miners. Fee depends on several factors and is changing over time.
+
+### via Clove API
+
+The easiest way to get current fee is to ask Clove API via `get_current_fee_per_kb` method. Clove API is a proxy between you and block explorer with a cache layer so you will not be exposed to requests limits in some block explorers.
+
+    from clove.network import Bitcoin
+
+    network = Bitcoin()
+    network.get_current_fee_per_kb()
+    0.00014827
+
+
+### via block explorer
+
+You can omit the Clove API and calculate fee directly from network's block explorer
+
+    from clove.network import Mooncoin
+
+    network = Mooncoin()
+    network.get_fee()
+    0.00200369
+
+some of those networks supports also counting fee from a given number of last transactions
+
+    network.get_fee(tx_limit=20)
+    0.00200362
+
+
+### Setting fee manually
+
+If you want to suggest your own fee value you can do this directly by setting `transaction.fee`.
+
+    transaction.fee = 0.005
+    transaction.add_fee()
+    transaction.sign(SOME_WALLET)
+
+
+## Publishing custom transactions
+
+Even if you have non atomic swap transaction you can still use clove for publishing.
+
+    network.broadcast_transaction(raw_transaction='010000000184a38a4e8743964249665fb241fbd3...35b')
```

Comment:
It should be `publish` instead of `broadcast_transaction`. Ethereum based networks don't have `broadcast_transaction`. Also the name of the argument differs between ethereum and bitcoin based networks (`transaction` and `raw_transaction`) so we can just skip it